src/main/java/io/github/mountainrange/mule/Player.java

```diff
@@ -1,25 +1,42 @@
 package io.github.mountainrange.mule;
 
+import java.util.EnumMap;
+import java.util.Objects;
+
+import io.github.mountainrange.mule.enums.Race;
+import io.github.mountainrange.mule.enums.ResourceType;
 import javafx.scene.paint.Color;
 
 /**
- * Created by white on 9/16/2015.
+ * Container that holds information about a particular player.
  */
 public class Player {
 
 	public static final String[] DEFAULT_NAME = {"Player 1", "Player 2", "Player 3", "Player 4"};
-	public static final Config.Race[] DEFAULT_RACE = { Config.Race.FOLD, Config.Race.FAULT_BLOCK,
-			Config.Race.DOME, Config.Race.VOLCANIC };
+	public static final Race[] DEFAULT_RACE = { Race.FOLD, Race.FAULT_BLOCK,
+			Race.DOME, Race.VOLCANIC, Race.PLATEAU };
 	public static final Color[] DEFAULT_COLOR = { Color.RED, Color.BLUE, Color.GREEN, Color.ORANGE };
 
+	private Race race;
+	private Color color = Color.BLACK;
+
 	private final int id;
 	private String name;
-	private Config.Race race;
-	private Color color = Color.BLACK;
 
-	public Player(int id, String name, Config.Race race, Color color) {
+	private int money;
```

Comment:
Money is always an integer in the original MULE game, so it should be fine. Eg prices in auctions only go up one at a time, and they don't have concepts like interest or anything.